docs/In-Band Manageability User Guide - ThingsBoard.md

```diff
@@ -1180,8 +1180,35 @@ dashboard to trigger a Device Reboot.
 Now on the pop-up window shows up, click the `Send` button on the box
 titled **`Decommission Device`**.
 
-### Query Commands
+### Query Command
 
+The Intel® In-Band Manageability provides a way to query attribute information on either the Host, Edge Device, or Nodes.
+
+To trigger a **Trigger Query**, follow the steps below:
+
+In order to query updates, click the **Trigger Query button** as seen below.
+
+<img src="media/In-Band Manageability User Guide - ThingsBoard/media/image65.png" style="width:5.78125in;height:3.59762in" />
+
```

Comment:
The pictures need to be consistent throughout the document.  All other pictures use a red box and this is using a black box.